src/utils.ts

```diff
@@ -48,6 +48,7 @@ export function defaultOptions(): Options {
     outputClientImpl: true,
     returnObservable: false,
     addGrpcMetadata: false,
+    addRestParameter: false,
```

Comment:
Maybe rename this to `addNestjsRestParameter`? Since it seems really specific to nestjs's controllers/clients.

Reply:
Totally agree! Changed it!